sigridci/sigridci/junit_format_report.py

```diff
@@ -19,8 +19,11 @@
 
 class JUnitFormatReport(Report):
 
+    def __init__(self, outputDir="sigrid-ci-output"):
```

Comment:
I'm a bit on the fence here, should we move this up into the Report class? Now we specify the default value in 3 subclasses. I think the other subclasses that write to stdout would just ignore this outputDir field. Alternatively, perhaps just put "sigrid-ci-output" as a string constant in the Report superclass and use that in the file-writing subclasses.

Reply:
Let's go for the latter. The downside of making it an optional parameter in the super constructor, is that it would then *also* need to be an optional argument in all of the child classes. Having it as a field will avoid that repetition.

Reply:
Something I'm not sure about: Do we want to use `PipelineSummaryReport` in this new entry point? Because that will return an exit code, but depending on the way you process this, you might not want it to do `sys.exit(1)`.